### PR TITLE
feat: Sync the authz tasks

### PR DESCRIPTION
**Pull request by tumido**
First part of: https://github.com/operate-first/apps/issues/211

Migrate content from:
- [x] https://github.com/open-infrastructure-labs/moc-cnv-sandbox/blob/master/roles/ocp/authz/tasks/main.yml, I've skipped:
  - `Group` definitions
  - `nad-view-user` `ClusterRole` and `self-provisioners`, `allow-view-nads` `ClusterRoleBindings` since it's related to the `approved-users` group, which was deleted
- [x] https://github.com/open-infrastructure-labs/moc-cnv-sandbox/blob/master/roles/ocp/tls/tasks/main.yml
- [x] https://github.com/open-infrastructure-labs/moc-cnv-sandbox/blob/master/roles/ocp/moc-sso/tasks/main.yml
- [x] https://github.com/open-infrastructure-labs/moc-cnv-sandbox/blob/master/roles/ocp/registry/tasks/main.yml

### CODE REVIEW

File: cluster-scope/base/imageregistryconfigs/cluster/config.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: imageregistry.operator.openshift.io/v1
+kind: Config
+metadata:
+  name: cluster
```
Comment (HumairAK) [inline cluster-scope/base/imageregistryconfigs/cluster/config.yaml:5]: This looks very different from the one in live: 

```yaml
apiVersion: operator.openshift.io/v1
kind: Config
metadata:
  annotations:
    release.openshift.io/create-only: "true"
  name: cluster
spec:
  managementState: Managed
status: {}
```

Comment (tumido) [reply cluster-scope/base/imageregistryconfigs/cluster/config.yaml:5]: that's a different `Config` kind, see the API group:
`operator.openshift.io/v1`'s`Config`  vs `imageregistry.operator.openshift.io`'s `Config`.

you'd need to query it as:
```
oc get configs.imageregistry.operator.openshift.io cluster
```

It confused me as well.

Comment (HumairAK) [reply cluster-scope/base/imageregistryconfigs/cluster/config.yaml:5]: gotcha thanks!

**Comment by HumairAK** [comment]
@tumido 
image

We could put some of these fields in, but I'm guessing `httpsecret` will need to be ignored in the cluster-resource app -- wdyt?

**Pull request merged**